notebooks/00-release-28.0.0/01-query-functions-window.ipynb

```diff
@@ -556,6 +556,25 @@
     "display.sql(req)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "fab5b831-69a8-4215-9b97-f7ce62610850",
+   "metadata": {},
+   "source": [
+    "## Cleanup\n",
+    "Run the following cell to remove the objects created within this notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8947babc-0798-465d-8eed-b9facca064d0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Drop datasource: [{druid.datasources.drop('example-wikipedia-windows')}]\")"
```

Comment:
Should this be the correct form of this going forward? Other notebooks just have the druid.datasources.drop call.
Or should this function output something itself?